packages/editor-ui/src/stores/cloudPlan.store.ts

```diff
@@ -0,0 +1,61 @@
+import { computed, reactive } from 'vue';
+import { defineStore } from 'pinia';
+import type { CloudPlanState } from '@/Interface';
+import { useRootStore } from '@/stores/n8nRoot.store';
+import { useSettingsStore } from '@/stores/settings.store';
+import { useUsersStore } from '@/stores/users.store';
+import { getCurrentPlan } from '@/api/cloudPlans';
+
+const DEFAULT_STATE: CloudPlanState = {
+	data: {
+		planId: 0,
+		monthlyExecutionsLimit: 0,
+		activeWorkflowsLimit: 0,
+		credentialsLimit: 0,
+		isActive: false,
+		displayName: '',
+		expirationDate: '',
+		metadata: {
+			version: 'v1',
+			group: 'opt-in',
+			slug: 'trial-1',
+		},
+		usage: {
+			executions: 0,
+			activeWorkflows: 0,
+		},
+	},
+};
+
+export const useCloudPlanStore = defineStore('cloudPlan', () => {
+	const rootStore = useRootStore();
+	const settingsStore = useSettingsStore();
+	const usersStore = useUsersStore();
+
+	const state = reactive<CloudPlanState>(DEFAULT_STATE);
+
+	const setData = (data: CloudPlanState['data']) => {
+		state.data = data;
+	};
+
+	const userIsTrialing = computed(() => state.data?.metadata.slug.includes('trial'));
```

Comment:
I would still use group here... I would expect that to be more consistent than slug

Reply:
But we can have multiple plans with group opt-in, not only the trial one?

Reply:
I guess we can do if it has expiration date and group is opt in

Reply:
that, or the plan id = 0 you had suggested before

Reply:
Or if it includes the trial key in the metadata

Reply:
I see the confusion.. trial belongs to the trial group